test/01_basic/cibuildwheel_test.py

```diff
@@ -0,0 +1,50 @@
+import subprocess, sys, os
+from glob import glob
+project_dir = os.path.dirname(__file__)
+sys.path.append(os.path.join(os.path.dirname(project_dir), 'shared'))
```

Comment:
Could we set this in the test driver, since it's (by definition) shared by all tests anyway? That's one less line to copy when adding a new test/one less line to change in every test file if there's a bug.

Adding (or appending to) the `PYTHONPATH` environment variable could work, here?

Reply:
Hm, yes I agree. PYTHONPATH is a good idea! I'll look at changing that.